cogs/fun.py

```diff
@@ -214,6 +215,39 @@ async def slot(self, ctx: Context[BotT]):
         else:
             await ctx.send(f"{slotmachine} No match, you lost 😢")
 
-
+    @commands.command()
+    async def dice(self,ctx:Context[BotT]):
+        """ Dice game. Good luck """
+        bot_dice=(random.randint(1, 6))
+        player_dice=(random.randint(1, 6))
+        await ctx.send(f"My dice   🎲 {bot_dice}\nYour dice 🎲 {player_dice}")
+        if bot_dice>player_dice:
+            await ctx.send('I won,try again!')
+        elif bot_dice==player_dice:
+            await ctx.send('Its a tie!')
+        else:
+            await ctx.send('Congrats,you won🎉!')
+
+    @commands.command(aliases=['colors'])
+    async def roula(self,ctx:Context[BotT]):
+        """ Colors roulette """
+        colors= ['blue','red','green','yellow']
+        message = (ctx.message.content.lstrip('$roula').split()[0])
+        print(message)
+        if not(message.lower() in colors):
+            await ctx.send('Please give correct color')
+        else:
+            chosen_color =random.choice(colors)
+            print(chosen_color.upper())
+
+            await ctx.send(f"Spinning 🔵🔴🟢🟡")
+            time.sleep(1)
```

Comment:
Using `time.sleep()` in async does not scale good when it comes to discord.py code and having possibly multiple servers. It does the job, however by the time the code executes it, everything on the bot stops for that one second. It is recommended to rather use `await asyncio.sleep(1)` to make sure that only that part of the code sleeps one 1 second while the rest of the code functions as usual without being blocked.